README.md

```diff
@@ -37,7 +37,7 @@ export default {
 }
 ```
 
-### \#set(key, value)
+### \#set(key, value, options)
```

Comment:
Thanks for this. While you're updating it could you please update the README to reflect `#set` -> `#setCookie` and same for `#get`?